bindings/guile/test/srfi64-extras.scm

```diff
@@ -22,28 +22,25 @@
 
 (export gnc:test-runner)
 (define (gnc:test-runner)
-  (let ((runner (test-runner-null))
-        (num-passed 0)
-        (num-failed 0))
+  (let ((runner (test-runner-null)))
     (test-runner-on-test-end! runner
       (lambda (runner)
         (format #t "[~a] line:~a, test: ~a\n"
                 (test-result-ref runner 'result-kind)
                 (test-result-ref runner 'source-line)
                 (test-runner-test-name runner))
         (case (test-result-kind runner)
-          ((pass xpass) (set! num-passed (1+ num-passed)))
           ((fail xfail)
            (if (test-result-ref runner 'expected-value)
                (format #t "~a\n -> expected: ~s\n -> obtained: ~s\n"
                        (string-join (test-runner-group-path runner) "/")
                        (test-result-ref runner 'expected-value)
-                       (test-result-ref runner 'actual-value)))
-           (set! num-failed (1+ num-failed)))
+                       (test-result-ref runner 'actual-value))))
           (else #t))))
     (test-runner-on-final! runner
       (lambda (runner)
         (format #t "Source:~a\npass = ~a, fail = ~a\n"
-                (test-result-ref runner 'source-file) num-passed num-failed)
-        (zero? num-failed)))
+                (test-result-ref runner 'source-file)
+                (test-runner-pass-count runner)
+                (test-runner-fail-count runner))))
```

Comment:
This should be two commits, the first to convert from using `num-passed` and `num-failed` to using the runner's stored counts, the second--which is what addresses the SRFI64 API change--removes the lambda's return.